docs/section-1/version-control.md

```diff
@@ -13,7 +13,7 @@ VS Code is the text editor we will be using to write and run our code, and shoul
 !!! note 
     If you look at the tutorial above, they mention the "Git: Clone" command from the Command Palette (⇧⌘P). However, it's good practice to use "Git: Clone(Recursive)" instead of just "Git(Clone)" because cloning the repo recursively also clones all the defined submodules in the repo as well.
 
-Navigate to this file in the repo (`docs/section-2/version-control.md`) and add your name to the list below.
+Navigate to this file in the repo (`docs/section-2/version-control.md`) and add your name to the list below, after which please also add your GitHub username to the "resources" (`docs/resources.md`) page.
```

Comment:
Link the google doc instead of putting it all on a public website. The google doc should have restricted access to only members on the team.

Reply:
Done.